src/plugin/protocol-htmlunit/src/java/org/apache/nutch/protocol/htmlunit/HttpResponse.java

```diff
@@ -272,6 +288,8 @@ public HttpResponse(HttpBase http, URL url, CrawlDatum datum)
         }
       }
 
+    }catch(Exception e) {
```

Comment:
The plugin must throw an exception when something goes wrong so that the calling instance (usually the Fetcher) can handle the exception. You need to catch the SSL-related exceptions (when initializing the trust manager) and rethrow them as ProtocolException.